src/OrleansProviders/Streams/Common/PooledCache/PooledQueueCache.cs

```diff
@@ -25,7 +25,6 @@ namespace Orleans.Providers.Streams.Common
     /// <typeparam name="TQueueMessage">Queue specific data</typeparam>
     /// <typeparam name="TCachedMessage">Tightly packed cached structure.  Should only contain value types.</typeparam>
     public class PooledQueueCache<TQueueMessage, TCachedMessage>
-        where TQueueMessage : class
```

Comment:
Constraint was not necessary, and I needed to use a structure for the TQueueMessage.  Constraint was originally added because type needed to be allowed to be null, but that was a very early iteration.  No reason for it now.